docs/faqs/using_floydhub.md

```diff
@@ -73,3 +73,16 @@ We have only a datacenter that is located in Oregon, USA.
 ![Oregon, USA map](../img/faqs_using_fh/Oregon.png)
 
 We have plans to add new datacenters in Europe and Asia in the future. Let us know if you need that for your projects.
+
+### How can I automatize my script to recognized that is running on FloydHub?
```

Comment:
`How can I automate my script to recognize...`

Reply:
Thanks!